.pipelines/.vsts-vhd-builder-windows.yaml

```diff
@@ -13,6 +15,23 @@
 # - SUBSCRIPTION_ID - Subscription ID used by the pipeline
 # - TENANT_ID - Tenant Id used for service principal login
 
+# Two modes in prioritized order
+# sigMode: Shared Image Gallery mode
+#    This modes is activated when mode 1 is not activated and by setting env var SIG_GALLERY_NAME/SIG_IMAGE_NAME/SIG_IMAGE_VERSION.
+#    It is similar to mode 1 except it will create a gen1 image and the SIG destination is specified by the provided env var.
+#    This is good for quick testing purpose as you can use the SIG image as soon as the build is completed.
+#    steps:
+#    - Build VHD to a SIG gallery as specified by env var SIG_GALLERY_NAME/SIG_IMAGE_NAME/SIG_IMAGE_VERSION
+#
+# default mode: Hyper-Generation V1 +VHD
+#    This default mode generates VHD and store it in a classic storage account. This is mainly for publishing a new VM image to Platform Image Repository(PIR) and Share Image Gallery.
+#    steps:
+#    - create an VHD in Packer to normal storage account
```

Comment:
What is the normal storage account and Packer storage account?

Reply:
copy and paste from [Linux pipeline](https://github.com/Azure/AgentBaker/blob/master/.pipelines/.vsts-vhd-builder.yaml#L22). `normal storage account` equals `Packer storage account`.